backend/src/main/java/com/swe573/socialhub/controller/UserController.java

```diff
@@ -65,8 +65,8 @@ public UserDto getUser(Principal principal, @PathVariable String userId) {
         }
     }
 
-    @DeleteMapping("/user/{userId}")
```

Comment:
I coded the admin dashboard according to @DeleteMapping code, why did we change it to PostMapping?